reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md

```diff
@@ -34,6 +34,29 @@ In this release, we're introducing a number of new features, including:
 To see a full list of features and fixes, check out
 our detailed [changelogs](https://github.com/PowerShell/PowerShell/releases) on GitHub.
 
+## Where can I Install PowerShell?
+
+PowerShell 7 supports the following operating systems on x64, including:
+
+TODO: have Travis validate
+
+Windows 7, 8.1, and 10
```

Comment:
Since Windows 7 is no longer supported - is it useful to mention that OS here? You could either just remove it, OR, say that it'll work but is not supported.